stdlib/private/StdlibUnittest/StdlibUnittest.swift.gyb

```diff
@@ -803,7 +814,10 @@ public func runAllTests() {
 "                        Useful for running under a debugger.\n" +
 "--stdlib-unittest-filter FILTER-STRING\n" +
 "                        only run tests whose names contain FILTER-STRING as\n" +
-"                        a substring."
+"                        a substring.\n" + 
+"--env STRING\n" +
```

Comment:
Could you just pass through all arguments (that don't start with `--stdlib-unittest`) to the child process?  This is how it was intended to be implemented.